wandb/sdk/backend/backend.py

```diff
@@ -65,7 +64,7 @@ def __init__(
         mailbox: Mailbox,
         settings: Optional[Settings] = None,
         log_level: Optional[int] = None,
-        manager: Optional[_Manager] = None,
+        service: "Optional[service_connection.ServiceConnection]" = None,
```

Comment:
if it is just the connection to the service, wouldn't something like `connection` or `service_connection`, `service_connector` be more correct?

Reply:
The subtle reason is that this class should abstract away the communication details. So it's not a *connection* to the service; it *is* the service. `service.teardown()` = "tear down the service", `service.send_record()` = "send a record to the service", etc. Saying "tear down the service connection" wouldn't be quite right (since the service is getting torn down, not the connection).

I wanted to call this class `Service`, but there already is a class by that name. The variable name `service_connection` is longer than `service` and conflicts with the import.

Ideally, `ServiceConnection` would be `Service`, and what's currently called `Service` would be `ServiceProcess`.

Reply:
got it! ok we can rename things in follow up clean up.. thanks